docs/validator/staking-overview.md

```diff
@@ -8,6 +8,10 @@ sidebar_label: Orientation
 
 This section introduces you to the validation process and some additional resources.
 
+NEAR Protocol is a Proof-of-Stake (PoS) blockchain, and validators are nodes that manage any new transactions, adding them to new blocks. Every new block provides revenues for the validator in form of NEAR tokens, which are sourced from transaction fees and protocol rewards. In order to become a validator, a node must stake a certain amount of NEAR tokens, which are used as a collateral against "equivocation", which means the malicious creation of multiple rival blocks at the same time. The tokens at stake cannot be spent until the validation cycle, called epoch, is over.
```

Comment:
A couple problems that I see here:
* Staking not only prevents double signing, but more importantly, prevents more damaging behaviors like committing invalid state transition.
* The tokens at stake are not returned at the end of the epoch but rather 3 epochs later (this is if the validator chooses to unstake).